### ISSUE TITLE
NC14.0.1 & quota_warning 1.3.0: "App is not compliant"

### ISSUE THREAD

**Issue by Sanookmakmak** [nextcloud/quota_warning#38]
`occ app:check-code quota_warning`

```
Analysing /var/www/html/apps/quota_warning/lib/CheckQuota.php
 3 errors
    line  321: OC_Util - Static method of private class must not be called
    line  322: OC_Util - Static method of private class must not be called
    line  323: OC_Helper - Static method of private class must not be called
Invalid appinfo.xml file found: Element 'name': This element is not expected. Expected is ( id ).

App is not compliant
```

**Comment by jcklpe** [comment]
Just wanted to confirm that this was an issue as of 14.03

**Comment by nickvergessen** [comment]
The app still works and can be installed.
This app will also always have the first 3messages, due to the internal information it needs.

**Comment by nickvergessen** [comment]
The invalid appinfo.xml has been fixed with 9f5124c437619fdb4e3f9d2b71376e1f9b633da3

**Comment by jcklpe** [comment]
Thanks!
